database/databaseOverlay/overlay.go

```diff
@@ -124,6 +125,9 @@ type Overlay struct {
 
 	BitcoinAnchorRecordPublicKeys  []interfaces.Verifier
 	EthereumAnchorRecordPublicKeys []interfaces.Verifier
+
+	// We need access to the state to be able emit anchor events
+	parentState events.StateEventServices
```

Comment:
Still looking at all the other changes to understand this - but my first impression is that it's strange to have an events dependency on the DB overlay @Emyrk thoughts?

Reply:
Yea, the anchors only live in the db. This is a bit strange, and I think to resolve it would require some anchor refactoring.

Reply:
spoke about this - we'll try to do something cleaner during refactor - but will merge this.